seqr/utils/elasticsearch/es_search.py

```diff
@@ -1280,6 +1284,41 @@ def parse_variant_id(cls, variant_id):
             raise ValueError('Invalid variant id')
         return var_fields[0].lstrip('chr'), int(var_fields[1]), var_fields[2], var_fields[3]
 
+    def _get_quality_filters_by_family(self, quality_filter):
```

Comment:
this basically did not change, it just got copy-pasted up and replaced the arguments with attributes from `self` (so instead of taking a `samples_by_family_index` argument, it uses `self. samples_by_family_index`). It does not need to be really reviewed